tikv-configuration-file.md

```diff
@@ -1511,6 +1523,18 @@ Configuration items related to `rocksdb.defaultcf`, `rocksdb.writecf`, and `rock
 + Default value: `"128MB"`
 + Unit: KB|MB|GB
 
+### `format-version`
```

Comment:
```suggestion
### `format-version` <span class="version-mark">New in v6.5.0</span>
```

Please update the release-6.5.0.md:
https://github.com/pingcap/docs/blob/1db43f06719ac567cd848e2e8f8d7616c8a90a38/releases/release-6.5.0.md?plain=1#L340-L348

Reply:
There're plenty of new configurations before 7.1 that are not documented. Let's stick to the 7.1 issues for now.